test/test_pretty_format.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import unittest
+from textwrap import dedent
+
+import orjson
+
+
+class DictPrettyFormatTests(unittest.TestCase):
+    def _strip_dedent(self, text):
+        return dedent(text).strip()
+
+    def _assert_equal_str(self, a, b):
```

Comment:
It's confused that this handles arbitrary input when it's a greenfield module where `a` is always `bytes` and `b` always `str`. This should compare `bytes` anyway since that is the output of orjson and it's serialized data.